README.md

```diff
@@ -1,49 +1,92 @@
+npm run start:dev
 
-### Steps to run locally - new method
-The new method is faster and easier to run in debug mode.
-* `./scripts/generate-dev-environment.sh local` - this will create a local `.env` file.
-* Repeat the same steps above.
-* Run the app with: `npm run start:dev`
-* Go to: http://localhost:3000 -> Note the diff port number.
+```
+Open application in browser: 
+- http://localhost:3000
+
+#### Local Docker
+If you have the backend services running locally in docker, you can connect directly to them.
+
+```
+ npm install && npm run compile
+
+ pay local launch admin
+ ./scripts/generate-dev-environment.sh local
+
+ npm run start:dev
+```
+
+Open application in browser: 
+- http://localhost:3000
+
+##### Log output
+When using Docker, you can view log out with the following command:
+```
+docker logs -f selfservice
+```
 
 #### Run in debug mode
-* You need to make sure the app runs locally first by using the instructions above.
+* You need to make sure the app runs locally first using the steps in the [Running](#running) section.
 * In VSCode, go to the `Debug` view - `CMD + shift + D`.
-* Run the launch config `nodemon` by selecting it and pressing the green play button.
+* From the **Run** toolbar, select tne launch config `Self Service`.
+* Add breakpoints to any file you want to debug - click in the left hand column and a red dot will appear.
+* Press The `green play` button (`F5` MacOS):
     * This will run the app in debug mode.
-    * Add breakpoints.
-    * The app will automatically restart when you make a code change.
+    * Uses `nodemon` so it will automatically restart on code changes.
+
+
+
+#### Watching for changes
+
+You shouldn’t need to restart the app to see changes you make.
+
+We use [nodemon](https://github.com/remy/nodemon) which watches for changes to files and restarts the node process.
+
+If you’re making changes to client-side JS or Sass files (anything within [`/browsered/`](https://github.com/alphagov/pay-selfservice/tree/BAU-update-README-to-explain-livereload/app/browsered) or [`/assets/`](https://github.com/alphagov/pay-selfservice/tree/BAU-update-README-to-explain-livereload/app/assets)) then running `npm run watch-live-reload` will watch for changes and recompile. Nodemon does not do anything here as that’s not necessary. If you install the [livereload browser plugin](http://livereload.com/extensions/) then it will refresh your page once the assets have been compiled to the `/public` folder.
 
 ## Running tests
 
 #### To run mocha tests
 ```
 npm run compile && npm test
 ```
+#### Run tests in debug mode
+
+##### IMPORTANT NOTE - some tests do not work in debug mode
+* Some integration tests do not work in debug mode.  This is because the tests are dependent on other tests running before hand.
+* This will need do be looked into the future.
+* Nevertheless, it is still useful to debug tests that do work in debug mode.
+
+##### Run tests in debug mode
```

Comment:
This is same as the section heading. Can this be specific ? something like `Debug using Visual Studio Code:`

Reply:
Done